extension.js

```diff
@@ -128,6 +129,11 @@ class ReferenceProvider {
         if (!newUriPath.match(/\.js$/i)) newUriPath += '.js';
 
         const newUri = vscode.Uri.file(newUriPath);
+
+        if (!fs.existsSync(newUri.fsPath)) {
```

Comment:
Thank you for the information. I don't think it is critical, but why add a few ms to the critical path? I suggest to leave at known issue and report on vscode.